cli.go

```diff
@@ -6,10 +6,19 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"log"
 )
 
 var discords []any
 
+func isAllowedClient(client string) bool {
+	switch client {
+	case "", "default", "stable", "ptb", "canary":
+		return true
+	default:
+		return false
+	}
+}
```

Comment:
this function needs some room to breathe
```suggestion
}

```